Jenkinsfile

```diff
@@ -351,8 +351,6 @@ pipeline {
                         checkPatch user: GITHUB_USER_USR,
                                    password: GITHUB_USER_PSW,
                                    ignored_files: "src/control/vendor/*:src/include/daos/*.pb-c.h:src/common/*.pb-c.[ch]:src/mgmt/*.pb-c.[ch]:src/iosrv/*.pb-c.[ch]:src/security/*.pb-c.[ch]:*.crt:*.pem:*_test.go:src/cart/_structures_from_macros_.h"
-                        sh label: "codespell",
-			   script: "codespell --ignore-words-list nd,uint,ths,ba,creat,te,cas,mapp,pres,crashers,dout,tre,reord,mimick,cloneable,keypair,bject,tread --builtin clear --skip *.png,*.PNG,*.pyc,src/rdb/raft/*,src/control/vendor/*,RSA.golden `git ls-tree --full-tree --name-only HEAD`"
```

Comment:
You don't want to have codespell always checking?

Reply:
I do, but not using this technique.  In fact we already have codespell working on PRs, and reporting into tickets.

This line in the Jenkinsfile would check the entire tree which found a number of issues, however when doing so it would stop the entire build at the checkpatch stage, and would be really difficult to find the cause so the Jenkinsfile part of this PR was useful to find the issues but not good enough to be a part or our ongoing CI process.

Reply:
Ahh.  I see.